include/LLVMSPIRVExtensions.inc

```diff
@@ -43,7 +43,7 @@ EXT(SPV_INTEL_variable_length_array)
 EXT(SPV_INTEL_fp_fast_math_mode)
 EXT(SPV_INTEL_fpga_cluster_attributes)
 EXT(SPV_INTEL_loop_fuse)
-EXT(SPV_INTEL_long_constant_composite)
```

Comment:
the renaming is a breaking change
please first change SPIRVReader in release branches to accept both SPV_INTEL_long_constant_composite and SPV_INTEL_long_composites, then wait for a graceful period and only then proceed with the renaming in ToT

Reply:
Gotcha, thanks!